gobblin-utility/src/main/java/org/apache/gobblin/util/AvroUtils.java

```diff
@@ -95,7 +95,7 @@ public class AvroUtils {
 
   public static final String AVRO_SUFFIX = ".avro";
 
-  private static final String SCHEMA_CREATION_TIME_KEY = "CreatedOn";
+  public static final String SCHEMA_CREATION_TIME_KEY = "CreatedOn";
```

Comment:
Why do we do this change, I think AvroUtils handles all the operation related to this prop?

Reply:
AvroSchemaUtil.convert in li-iceberg returns a shaded avro schema object so I cannot use AvroUtils.setProperty. Therefore I need to set the SCHEMA_CREATION_TIME_KEY directly on the shaded object and its better to use this Key from AvroUtils rather than create a new Key